Source/WebCore/platform/ios/LocalCurrentTraitCollection.mm

```diff
@@ -29,23 +29,20 @@
 
 #import <pal/ios/UIKitSoftLink.h>
 
-#if HAVE(OS_DARK_MODE_SUPPORT)
-#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/LocalCurrentTraitCollectionAdditions.mm>)
-#import <WebKitAdditions/LocalCurrentTraitCollectionAdditions.mm>
-#else
 namespace WebCore {
 
+#if HAVE(OS_DARK_MODE_SUPPORT)
 static UITraitCollection *adjustedTraitCollection(UITraitCollection *traitCollection)
 {
+#if PLATFORM(VISION)
+    // Use the iPad idiom instead of the Reality idiom, as light mode colors are effectively dark mode
```

Comment:
This comment may be inaccurate now; I think *this* part is fixed, but some critical system colors are still transparent.

Reply:
Will update the comment to:

```
Use the iPad idiom instead of the Reality idiom, since some system colors are transparent in the Reality idiom, and are not web-compatible.
```